static/js/recent_topics.js

```diff
@@ -667,10 +672,20 @@ export function change_focused_element(e, input_key) {
                 }
                 break;
             case "vim_down":
+                if (is_focus_at_last_table_row()) {
```

Comment:
Maybe also worth a comment here describing briefly the `kkkk` issue background, since this is a little non-intuitive.

Reply:
Done.